keep/api/core/db.py

```
@@ -1296,50 +1297,50 @@
                 # SQLite version - using JSON
                 incidents_subquery = (
                     session.query(
-                        AlertToIncident.alert_id,
+                        LastAlertToIncident.fingerprint,
```

Ruff

`LastAlertToIncident` may be undefined, or defined from star imports

(F405)